buildscripts/update-lobster.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# this script pulls the evergreen-ci/lobster repo and copies the files so that
+# they can be served by logkeeper
+# it should only be run from the base logkeeper directory
+set -o errexit
```

Comment:
it might be helpful to also `set -o xtrace`

Reply:
done